site/quickstarts/index.html

```diff
@@ -0,0 +1,178 @@
+---
+layout: doc-landing
```

Comment:
Are we going to use this layout/header for all the docs pages? I think the answer is yes.

Reply:
Not necessarily. But for now, for the quick launch let's role with it and explore what that may look like down the road.